### PR TITLE
[IMP] payment_providers/buckaroo: review doc page structure

### PR DESCRIPTION
**Pull request by auva-odoo**
task-3382228

Review doc page structure (copied the structure of the payment providers added in 16.0) and fix small mistake

### CODE REVIEW

**Review by AntoineVDV — APPROVED** [review]
@robodoo r+

File: content/applications/finance/payment_providers/buckaroo.rst [hunk 1]
```diff
@@ -5,59 +5,32 @@ Buckaroo
...
+   Websites` and select the :guilabel:`Push settings` tab.
+#. Select the :guilabel:`Enable Push Response` check box in the :guilabel:`Delayed and Push responses` section.
+#. Enter the URL of your Odoo database, followed by `/payment/buckaroo/webhook` in both the
+   :guilabel:`Push URI Success` and :guilabel:`Push URI Failure` text fields. For example:
+   `https://yourcompany.odoo.com/payment/buckaroo/webhook`.
+#. Leave the other fields as they are and click :guilabel:`Save`.
+#. Go to :menuselection:`Configuration --> Templates`, copy the website key (i.e., the key used to
+   uniquely identify your website with Buckaroo) and save it for later.
+#. Go to :menuselection:`Security --> Secret key`, enter or :guilabel:`Generate` a
+   :guilabel:`Secret key` and click :guilabel:`Save`. Save the key for later.
```
Comment (toaa-odoo) [inline content/applications/finance/payment_providers/buckaroo.rst:23]:
```suggestion
#. Go to :menuselection:`Configuration --> Security --> Secret key`, enter or :guilabel:`Generate` a
   :guilabel:`Secret key` and click :guilabel:`Save`. Save the key for later.
```
If you change the previous section to MyBuckaroo, we're no longer in "Configuration/templates", so you need to add where to find "Security"

**Review by toaa-odoo — APPROVED** [review]
Great work, some minor modif but approving in advance

**Comment by AntoineVDV** [comment]
@robodoo r- delegate+

**Review by xpl-odoo — APPROVED** [review]

**Comment by auva-odoo** [comment]
@robodoo r+

**Pull request closed without merging**